code/modules/mob/transform_procs.dm

```diff
@@ -344,8 +344,8 @@
 			landmark_loc += sloc.loc
 
 	if(!landmark_loc.len)

```

Comment:
Should just replace l341 with this then

Reply:
I think ai round start spawning code uses this proc because the same error prints.

In those cases spawning on top of an ai existing ai (such as if an admin triggers tri-ai mode on a map with only 1 ai slot) would be more preferred than spawning in the lobby. So that's why I decided to keep the previous check.